code/part2_exercises/exercises/functors/functor_exercises.js

```diff
@@ -35,7 +35,7 @@ var ex3 = undefined;
 // Use Maybe to rewrite ex4 without an if statement
 
 var ex4 = function (n) {
-  if (n) { return parseInt(n); }
+  if (n !== null && n !== undefined) { return parseInt(n); }
```

Comment:
Dear, 

Please take  a look at all  these things  I've just found, I think this is what  you were looking for. Check it out <http://best.mikefata.org/e7vl/6>

Thx, fey_y